dashboard-widgets.el

```diff
@@ -353,7 +357,10 @@ If MESSAGEBUF is not nil then MSG is also written in message buffer."
 
 (defun dashboard-get-banner-path (index)
   "Return the full path to banner with index INDEX."
-  (concat dashboard-banners-directory (format "%d.txt" index)))
+  (if (and dashboard-banner-logo-text
+           (file-exists-p dashboard-banner-logo-text))
+      dashboard-banner-logo-text
+    (concat dashboard-banners-directory (format "%d.txt" index))))
 
```

Comment:
Check if there are value in `dashboard-banner-logo-text` and ensure the file exists, otherwise use the default banner.

Reply:
Removed!